assets/src/edit-story/components/inspector/karma/inspectorTabs.karma.js

```diff
@@ -134,5 +134,36 @@ describe('Inspector Tabs integration', () => {
         ).not.toBe(null);
       });
     });
+
+    it('should navigate to checklist tab after following "review checklist" button in dialog on publishing story', async () => {
+      fixture.events.click(fixture.editor.titleBar.publish);
+      // Ensure the debounced callback has taken effect.
+      await fixture.events.sleep(800);
+
+      const reviewButton = await fixture.screen.getByRole('button', {
+        name: /^Review Checklist$/,
+      });
+      await fixture.events.click(reviewButton);
+      await fixture.events.sleep(300);
+
+      // expect the checklist tab to be selected and nothing to be disabled
+      await waitFor(() => {
+        expect(
+          fixture.editor.inspector.checklistTab.getAttribute('aria-selected')
+        ).toBe('true');
+
+        expect(
+          fixture.editor.inspector.checklistPanel.recommended.getAttribute(
+            'disabled'
+          )
+        ).toBe(null);
```

Comment:
💭 maybe this should be `.not.toBeTrue()`? I'm just wondering if `disabled` could ever be `false`

Reply:
`disabled` is not present at all if it's not true here so null works better